release-notes/opensearch-release-notes-1.2.0.md

```diff
@@ -0,0 +1,528 @@
+* Update license and attributions for faiss addition ([#187](https://github.com/opensearch-project/k-NN/pull/187))
+* Update license headers ([#194](https://github.com/opensearch-project/k-NN/pull/194))
+* Update license headers in gradle files ([#201](https://github.com/opensearch-project/k-NN/pull/201))
+
+
+### Opensearch Trace Analytics
+* Refactor trace analytics UT and IT, sync main branch ([#107](https://github.com/opensearch-project/trace-analytics/pull/107))
+* Bump prismjs from 1.24.1 to 1.25.0 ([#137](https://github.com/opensearch-project/trace-analytics/pull/137))
+* Bump immer from 9.0.5 to 9.0.6 ([#136](https://github.com/opensearch-project/trace-analytics/pull/136))
+* Update data modal and enable CI ([#148](https://github.com/opensearch-project/trace-analytics/pull/148))
+* Add integration tests for observability backend plugin ([#180](https://github.com/opensearch-project/trace-analytics/pull/180))
+* Bump ansi-regex to 5.0.1 ([#241](https://github.com/opensearch-project/trace-analytics/pull/241))
+* Add support for codeowners to repo ([#244](https://github.com/opensearch-project/trace-analytics/pull/244))
+* Panels cypress test ([#256](https://github.com/opensearch-project/trace-analytics/pull/256))
+
+
+## DOCUMENTATION
+
+### Opensearch Alerting
+* Add release notes for 1.2.0.0 release ([#225](https://github.com/opensearch-project/alerting/pull/225))
+
+
+### Opensearch Anomaly Detection
+* Correct copyright notice; remove old copyright from ODFE ([#257](https://github.com/opensearch-project/anomaly-detection/pull/257))
+* Add DCO Check Workflow ([#273](https://github.com/opensearch-project/anomaly-detection/pull/273))
+
+
+### Opensearch Dashboards Reports
+* Update validation for observability notebooks integration ([#174](https://github.com/opensearch-project/dashboards-reports/pull/174))
+* Add dco and release drafter workflows ([#217](https://github.com/opensearch-project/dashboards-reports/pull/217))
+* update README notification section ([#216](https://github.com/opensearch-project/dashboards-reports/pull/216))
+
+
+### Opensearch Dashboards Visualizations
+* Update copyright notice in README and headers ([#35](https://github.com/opensearch-project/dashboards-visualizations/pull/35))
+
+
+### Opensearch Index Management
+* Automatically provides license header for new files ([#142](https://github.com/opensearch-project/index-management/pull/142))
+
+
+### Opensearch Job Scheduler
+* Add release notes for 1.2.0.0 ([#84](https://github.com/opensearch-project/job-scheduler/pull/84))
+
+
+### Opensearch k-NN
+* Add support for codeowners to repo ([#206](https://github.com/opensearch-project/k-NN/pull/206))
+
+
+### Opensearch Performance Analyzer
+* Update README.md ([#88](https://github.com/opensearch-project/performance-analyzer-rca/pull/88))
+
+
+### Opensearch SQL
+* Create 1.2 release notes ([#268](https://github.com/opensearch-project/sql/pull/268))
```

Comment:
Yo, I hear you like release notes in your release notes 😏

Reply:
Haha! Do we need to remove this?